spotbugs/src/main/java/edu/umd/cs/findbugs/TextUIBugReporter.java

```diff
@@ -240,6 +242,19 @@ protected void checkBugInstance(BugInstance bugInstance) {
         }
     }
 
+    public String getOutputTarget() {
+        return this.outputTarget;
```

Comment:
Why use `this` when there is no local variable to confuse it with?

Reply:
I don't see harm there, personally I prefer objects to be qualified as its easier to understand where its defined.